shared/reports/resources.py

```diff
@@ -633,6 +633,13 @@ def __init__(
     def from_chunks(cls, *args, **kwargs):
         return cls(*args, **kwargs)
 
+    @property
+    def size(self):
```

Comment:
What's the unit? Is it... number of lines? Size of chunks in bytes? If you know please add a comment with that info

Reply:
It's kind of a unitless number.  It's the summed number of characters in each chunk - and the chunk itself is newline-separated JSON.  So sorta bytes but not really?